packages/node/examples/fastify/index.js

```diff
@@ -0,0 +1,38 @@
+import Fastify from 'fastify';
+import readmeio from 'readmeio';
+
+const fastify = Fastify({
+  logger: true,
+});
+
+fastify.decorateRequest('readmeStartTime', '');
+fastify.addHook('onRequest', (request, reply, next) => {
```

Comment:
If you mark this callback as `async`, you can remove the use of `next` altogether. Fastify will do it under the hood.